### PR TITLE
[dbmanager] Fix inconsistent capitalization, HIG violations and unprofessional strings

### PR DESCRIPTION
**Pull request by nyalldawson**
Subject says it all...

### CODE REVIEW

File: python/plugins/db_manager/db_manager_plugin.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def initGui(self):
         else:
             self.iface.addPluginToMenu(QApplication.translate("DBManagerPlugin", "DB Manager"), self.action)
 
-        self.layerAction = QAction(QgsApplication.getThemeIcon('dbmanager.svg'), QApplication.translate("DBManagerPlugin", "Update Sql Layer"),
+        self.layerAction = QAction(QgsApplication.getThemeIcon('dbmanager.svg'), QApplication.translate("DBManagerPlugin", "Update Sql Layer…"),
```
Comment (nirvn) [inline python/plugins/db_manager/db_manager_plugin.py]: We should settle on either Sql or SQL through DB manager. 

You had another change with SQL being used: ` self.actionSqlWindow = self.menuDb.addAction(QIcon(":/db_manager/actions/sql_window"), self.tr("&SQL Window"),`. My vote goes to SQL :)

File: python/plugins/db_manager/db_plugins/oracle/plugin.py [hunk 2]
```diff
@@ -225,36 +225,36 @@ def registerDatabaseActions(self, mainWindow):
...
         action = QAction(QIcon(":/db_manager/actions/create_table"),
                          QApplication.translate(
-                             "DBManagerPlugin", "&Create table"), self)
+                             "DBManagerPlugin", "&Create Table…"), self)
         mainWindow.registerAction(action, QApplication.translate(
             "DBManagerPlugin", "&Table"), self.createTableActionSlot)
         action = QAction(QIcon(":/db_manager/actions/edit_table"),
                          QApplication.translate(
-                             "DBManagerPlugin", "&Edit table"), self)
+                             "DBManagerPlugin", "&Edit Eable…"), self)
```
Comment (nirvn) [inline python/plugins/db_manager/db_plugins/oracle/plugin.py]: Eable -> Table

File: python/plugins/db_manager/db_plugins/plugin.py [hunk 3]
```diff
@@ -175,7 +175,7 @@ def addConnectionActionSlot(self, item, action, parent):
     def removeActionSlot(self, item, action, parent):
         QApplication.restoreOverrideCursor()
         try:
-            res = QMessageBox.question(parent, QApplication.translate("DBManagerPlugin", "hey!"),
+            res = QMessageBox.question(parent, QApplication.translate("DBManagerPlugin", "DB Manager"),
```
Comment (nirvn) [inline python/plugins/db_manager/db_plugins/plugin.py]: Hey, good move! 😉

File: python/plugins/db_manager/db_manager.py [hunk 4]
```diff
@@ -448,8 +448,8 @@ def setupUi(self):
         sep.setVisible(False)
 
         self.actionImport = self.menuTable.addAction(QIcon(":/db_manager/actions/import"),
-                                                     self.tr("&Import layer/file"), self.importActionSlot)
-        self.actionExport = self.menuTable.addAction(QIcon(":/db_manager/actions/export"), self.tr("&Export to file"),
+                                                     self.tr("&Import Layer/file…"), self.importActionSlot)
```
Comment (DelazJ) [inline python/plugins/db_manager/db_manager.py]: Should it not be *File*, given https://github.com/qgis/QGIS/pull/6325/files#diff-abadc3d841d1da75bab0d7546cbc2e95R253?

File: python/plugins/db_manager/db_plugins/plugin.py [hunk 5]
```diff
@@ -308,34 +308,34 @@ def registerDatabaseActions(self, mainWindow):
...
+                         QApplication.translate("DBManagerPlugin", "&Create Table…"), self)
         mainWindow.registerAction(action, QApplication.translate("DBManagerPlugin", "&Table"),
                                   self.createTableActionSlot)
         action = QAction(QIcon(":/db_manager/actions/edit_table"),
-                         QApplication.translate("DBManagerPlugin", "&Edit table"), self)
+                         QApplication.translate("DBManagerPlugin", "&Edit Table…"), self)
         mainWindow.registerAction(action, QApplication.translate("DBManagerPlugin", "&Table"), self.editTableActionSlot)
         action = QAction(QIcon(":/db_manager/actions/del_table"),
-                         QApplication.translate("DBManagerPlugin", "&Delete table/view"), self)
+                         QApplication.translate("DBManagerPlugin", "&Delete Table/view…"), self)
```
Comment (DelazJ) [inline python/plugins/db_manager/db_plugins/plugin.py]: View?

File: python/plugins/db_manager/dlg_table_properties.py [hunk 6]
```diff
@@ -238,7 +238,7 @@ def currentConstraint(self):
         sel = self.viewConstraints.selectionModel()
         indexes = sel.selectedRows()
         if len(indexes) == 0:
-            QMessageBox.information(self, self.tr("DB Manager"), self.tr("Nothing selected"))
+            QMessageBox.information(self, self.tr("DB Manager"), self.tr("No constraints were selected"))
```
Comment (DelazJ) [inline python/plugins/db_manager/dlg_table_properties.py]: missing period

**Review by DelazJ — COMMENTED** [review]
Thanks, Nyall

**Pull request merged**